packages/form/components/Field.astro

```diff
@@ -95,4 +95,8 @@ const validatorAttributes: Record<string, string> = validators?.reduce(
 		color: red;
 		border-color: red;
 	}
+
+	div[data-required='true'] + label::before {
```

Comment:
I think it should be `data-validator-required` can you check?

Reply:
Sure, fixed!